src/Tooltip/TooltipDictionary.yaml

```
@@ -2021,7 +2020,7 @@ gitSyncRepoTab_rootFolder: |-
 
     New to Harness Git Experience? See the [Quickstart](https://developer.harness.io/docs/platform/Git-Experience/configure-git-experience-for-harness-entities).
 gitSyncRepoTab_isDefault: |-
-    **What is the default folder?** When a Pipeline in one repo needs to access a Connector/Secret/etc in another repo at runtime, the Pipeline will always use the Connector/Secret/etc in their **default folder**. Learn More](https://developer.harness.io/docs/platform/Git-Experience/manage-a-harness-pipeline-repo-using-git-experience)
+    **What is the default folder?** When a pipeline in one repo needs to access a connector/secret/etc. in another repo at runtime, the pipeline will always use the Connector/Secret/etc in their **default folder**. Learn More](https://developer.harness.io/docs/platform/Git-Experience/manage-a-harness-pipeline-repo-using-git-experience)
```

```suggestion
    **What is the default folder?** When a pipeline in one repo needs to access a resource such as a connector or secret in another repo at runtime, the pipeline always uses the resource in the **default folder** in the external repo. [Learn More](https://developer.harness.io/docs/platform/Git-Experience/manage-a-harness-pipeline-repo-using-git-experience)
```
Optional word tweaks, but the link is missing the opening square bracket.